maven/org.eclipse.emf.mwe2.target/org.eclipse.emf.mwe2.target.target

```diff
@@ -31,7 +31,7 @@
 <repository location="http://download.eclipse.org/modeling/m2t/xpand/updates/releases/R201605260315"/>
 </location>
 <location includeAllPlatforms="false" includeMode="planner" includeSource="true" type="InstallableUnit">
-<unit id="org.junit" version="0.0.0"/>
+<unit id="org.junit" version="1.0.1.v20140414-1359"/>
```

Comment:
Because we need this exact version and newer versions would be available?

Reply:
yes. the newer is for epl 2

Reply:
update pr. had adopted wrong one